stm32cube/stm32wbaxx/README

```diff
@@ -49,4 +48,7 @@ Patch List:
     Impacted files:
      drivers/include/Legacy/stm32_hal_legacy.h
 
+   *Including common interfaces of MAC only and BLE/COEXISTENCE APIs
+    -Added lib/stm32wba/hci/ll_intf_cmn.h
+
```

Comment:
This is the HAL README, please update the lib/stm32wba/hci/README instead, there is already a list of the files included.

Reply:
Done